driver/optmaindriver.ml

```diff
@@ -66,6 +66,7 @@ let main argv ppf ~flambda2 =
       ["-depend", Arg.Unit Makedepend.main_from_option,
        "<options> Compute dependencies \
         (use 'ocamlopt -depend -help' for details)"];
+    Clflags.Opt_flag_handler.set Flambda_backend_flags.opt_flag_handler;
```

Comment:
Do you need to do something like this in `native_toplevel/opttopmain.ml` as well?

Reply:
Probably, if we want the toplevel to honor optimization attributes. Added. Not sure how to test it, though?